Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/bugreporting/DebugLogFragmentTest.kt

```diff
@@ -71,7 +71,7 @@ class DebugLogFragmentTest : BaseUITest() {
     @Screenshot
     @Test
     fun capture_inactive_screenshot() {
-        launchFragmentInContainer2<DebugLogFragment>()
+        launchInEmptyActivity<DebugLogFragment>()
```

Comment:
How about making `launchFragmentInContainer2` launch in our EmptyActivity by default ?

Reply:
The documentation of `launchInMainActivity()` says

```
Note: In cases where screenshots are not required, it is better to use testing framework launcher.
This helper is convenient for screenshots when [BottomNavigationView] should be visible in the screenshot.
```

`launchFragmentInContainer2()` is the "testing framework launcher", right? 
So I thought we need: 
- `launchInEmptyActivity()` for Screenshot Tests
- `launchFragmentInContainer2()` for pure Instrumentation tests

Is that correct?

Reply:
`launchFragmentInContainer2`  is our version of `launchFragmentInContainer` -from testing API- 
what is `launchFragmentInContainer2` is  currently doing is passing our `AppTheme` . 
and it used actually for screenshots tests. 
I suggest that we change the implementation of `launchFragmentInContainer2` to call  internally `launchInMainActivity` and then tests using don’t have to know which activity is used , by doing that when the root issue is fixed in testing library we can remove the workaround and don't have to change it again everywhere , what do you think ?

Reply:
Could you please also revert the changes that was made before in some layouts to use `AppCompatImageView` instead of `ImageView`. it should unnecessary now ?

Reply:
Changed all `AppCompatImageView`s to `ImageView`.